core/domain/stats_services.py

```diff
@@ -29,6 +29,24 @@
 # Counts contributions from all versions.
 VERSION_ALL = 'all'
 
+def get_exploration_stats(exp_id, exp_version):
+    """Retrieves the ExplorationStatsModel and returns it in dict format.
+
+    Args:
+        exp_id: str. ID of the exploration.
+        exp_version: int. Version of the exploration.
+
+    Returns:
+        dict. The ExplorationStatsModel's dict format.
+    """
+    exploration_stats = get_exploration_stats_by_id(exp_id, exp_version)
+
+    if exploration_stats is None:
+        exploration_stats = stats_domain.ExplorationStats(
```

Comment:
Consider encapsulating this into stats_domain.ExplorationStats.create_default(). That way, if a new argument is added in the future, you don't have to modify the args all over the codebase.

Reply:
Done.